vsphere/datadog_checks/vsphere/config.py

```diff
@@ -157,6 +159,14 @@ def validate_config(self):
                 "integer between 1 and 4."
             )
 
+        all_valid_resource_types = list(MOR_TYPE_AS_STRING.values())
+        for resource_type in self.event_resource_filters:
+            if resource_type not in all_valid_resource_types:
```

Comment:
suggestion: we can convert the input to lowercase to relax the validation

Reply:
good idea!